frontend/integration-tests/tests/kubevirt/clone.vm.scenario.ts

```diff
@@ -3,6 +3,8 @@ import { execSync } from 'child_process';
 import { $, $$, browser, ExpectedConditions as until } from 'protractor';
 import * as _ from 'lodash';
 
+import { emptyStr } from './mocks';
```

Comment:
why here?

Reply:
Forgot change L152 in this file, updated